Jenkinsfile

```diff
@@ -5,16 +5,22 @@ pipeline {
   agent any
 
   stages {
+    stage('Build') {
+      script {
+        // package.json should not have private dependencies
+        // we should be able to install dependencies without credentials
+        docker.image('node:12-alpine').inside() {
```

Comment:
why not use Dockerfile.ci file like other projects have?

Reply:
discussed offline. will leave it as is.